homeassistant/components/motioneye/camera.py

```diff
@@ -203,7 +202,7 @@ def _get_mjpeg_camera_properties_for_camera(
                 streaming_url = self._client.get_camera_stream_url(camera)
 
         return {
-            CONF_NAME: camera[KEY_NAME],
+            CONF_NAME: None,
```

Comment:
Instead of this, should the camera unconditionally set its name to `None`?

Reply:
You mean `_attr_name`?

Reply:
yes, exactly

Reply:
The fact is that in the Mjpegcamera, `_attr_name` is set to name, that's why I put name as None, so `_attr_name` will be None

Reply:
I understand, but my point is that you can just remove this line `CONF_NAME: camera[KEY_NAME],` and hard code _attr_name to None. Or do I misunderstand?

Reply:
Haven't thought of that, thanks